chronicles.nim

```diff
@@ -228,10 +238,13 @@ macro logIMPL(Stream: typed,
     # in `log_output` to kick in.
     let recordRef = if recordArity == 1: record
                     else: newTree(nnkBracketExpr, record, newLit(i))
+    var filename = lineInfo.filename & ":" & $lineInfo.line
     code.add quote do:
       initLogRecord(`recordRef`, LogLevel(`severity`),
                     `topicsNode`, `eventName`)
       setFirstProperty(`recordRef`, "thread", `threadId`)
+      if `lineNumbersNN`:
```

Comment:
This doesn't need to be in the generated code. Put the `if` outside of the `quote` block:

``` nim
if useLineNumbers:
  code.add newCall("setProperty", recordRef, ...)
```